sigproc/inc/WireCellSigProc/OmnibusSigProc.h

```diff
@@ -21,6 +22,7 @@ namespace WireCell {
                      const std::string& field_response = "FieldResponse",
                      double fine_time_offset = 0.0 * units::microsecond,
                      double coarse_time_offset = -8.0 * units::microsecond,
+                     const std::string& elecresponse_tn = "ColdElec",
```

Comment:
Just a general coding comment: Normally you want to avoid adding a parameter in the middle of an argument list as it will break all callers of the class.  Instead it is best added to the end (because it can have a default value).  But, in this particular case, we usually call with a default constructor and just take all these defaults as-is so no harm.  Some tests may call with non-default args, but if they compile then they won't care about this change.

Reply:
Thanks, I'll keep in mind for the next time